tests/RemoteStorageDB/CMakeLists.txt

```diff
@@ -3,5 +3,5 @@ link_directories(${CMAKE_BINARY_DIR}/lib)
 
 add_executable(Test_Mongo Test_mongo.cpp)
 target_include_directories(Test_Mongo PUBLIC ${CMAKE_SOURCE_DIR}/src ${CMAKE_SOURCE_DIR}/tests)
-target_link_libraries(Test_Mongo PUBLIC RemoteStorageDB TestUtils)
+target_link_libraries(Test_Mongo PUBLIC RemoteStorageDB TestUtils AccountData)
```

Comment:
seems not needed

Reply:
If you try and build test_mongo it is needed as account data is not linked.

Reply:
okay